### ISSUE TITLE
feat: Translate install list of flatpaks from `default-flatpaks` module into ISO action

### ISSUE THREAD

**Issue by fiftydinar** [blue-build/github-action#20]
Users would not need to wait for flatpaks to install once they boot into the system.
That installation step will be in the ISO installer screen instead.

Related issue:
https://github.com/ublue-os/isogenerator/issues/9

**Comment by fiftydinar** [comment]
I think that this issue can be closed, unless it can be useful for generating offline ISO?

**Comment by xynydev** [comment]
I don't know how the Flatpak inclusion works, or if that's even implemented yet. I'm also doubting / delaying the creation of an ISO action for a few reasons:
1. Upload limits would require some configuration of the server where the ISOs are uploaded by the user, so it wouldn't be that automatic.
2. A list of ISOs would also have to be configured, which also wouldn't be that automatic.
3. Both of those things would be / are handled by the upstream action, which we could just provide a guide for adding to ones repo.
4. The ISO action would only be useful for building on Fedora, other OSs that might be supported in the future for ISOs would have to have other mechanisms.
5. The only benefit an ISO action from us would provide is this Flatpak auto-conf feature, and while I'm pretty sure we could do a reasonably simple implementation that traverses the recipe to pick the Flatpaks to add to the ISO, it would be adding a new semantic meaning for the default-flatpaks module that isn't totally functionally related. Someone might want to include only a small set of Flatpaks in the ISO to keep file sizes smaller, for example, and install the rest from the internet post-install.

**Comment by fiftydinar** [comment]
You are right on all points imo, so I think that provided easy guide for making ISOs would be enough.
If Universal Blue adds support for online ISOs, we would update the guide with that option too to be automatic, like it was with old action.

**Comment by gmpinder** [comment]
So I looked at ublue's new ISO generator and it looks like something that we could add to the CLI for people to build one for themselves locally. We've already got drivers for running builds, we could support running their new image to create the ISO.

**Comment by xynydev** [comment]
That'd be cool! I was thinking of ISO generation for the planned GUI, but of course it would work with the CLI as well.
